### PR TITLE
Add Gradle Wrapper

### PR DESCRIPTION
**Pull request by yukiisbored**
Gradle wrapper is essential for a gradle project because it helps users
who doesn't have Gradle or Ant installed to build the project just with
the JDK without the hassle of installing gradle or ant.

This helps the users who aren't Java developers and just want to use
Loklak on their machine.

Fixes https://github.com/loklak/loklak_server/issues/897

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: gradlew.bat [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+echo.
+echo ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
+echo.
+echo Please set the JAVA_HOME variable in your environment to match the
+echo location of your Java installation.
+
+goto fail
+
+:findJavaFromJavaHome
```
Comment (sudheesh001) [inline gradlew.bat]: `^M` ?
Can you strip out these DOS line endings ?
```
:set ff=unix
:wq
```

Comment (yukiisbored) [inline gradlew.bat]: This is a batch file for Windows.

Comment (sudheesh001) [inline gradlew.bat]: Won't it work without the `^M` ? I need time until monday to test if this is working on a windows PC.

Comment (yukiisbored) [inline gradlew.bat]: @sudheesh001 It'll look weird on `notepad.exe`

Comment (yukiisbored) [inline gradlew.bat]: @sudheesh001 Have you confirmed this? I can't test it because I don't have a Windows machine or a Windows installation disc

Comment (sudheesh001) [inline gradlew.bat]: Just verified. Works, I'll go ahead and merge this. Thanks !

**Pull request merged**